bndtools.core/src/bndtools/editor/BndEditor.java

```diff
@@ -328,8 +329,9 @@ public IStatus runInUIThread(IProgressMonitor monitor) {
 					sourcePage.commit(false);
 				}
 
+				ResolverLogger logger = new ResolverLogger();
```

Comment:
Who closes this ResolverLogger? I have the feeling there are several code paths where it is quite unclear who, if anyone, closes the ResolverLoggers.

Reply:
you are right, the implementation was a bit too messy. I need to keep the logger till the End, so I can instruct it not to delete the logfile if the resolver ran in circles and needed canceling. 

The logger will will now be created for each resolve operation and will be closed when the wizard is disposed or the when we reresolve after adding optional bundles